simtools/corsika/__init__.py

```diff
@@ -0,0 +1,3 @@
+import logging
+
+logging.basicConfig(format='%(levelname)s::%(module)s(l%(lineno)s)::%(funcName)s::%(message)s')
```

Comment:
(I don't understand - but it is me!)

Reply:
I don't think this is needed. I will remove it.
I just copied this files from another directory.